src/applications/disability-benefits/wizard/pages/bdd.jsx

```diff
@@ -27,16 +28,11 @@ const alertContent = (
       file a BDD claim, but you can still begin the process of filing your claim
       on eBenefits.
     </p>
-    <a
-      href="https://www.ebenefits.va.gov/ebenefits/about/feature?feature=disability-compensation"
```

Comment:
I figured if we're going to link to this in multiple places, we may as well make it a constant.